crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -53,24 +61,42 @@ struct CreditCardDetails {
     card_code: Option<masking::Secret<String>>,
 }
 
-#[derive(Serialize, Deserialize, PartialEq, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
 struct BankAccountDetails {
     account_number: masking::Secret<String>,
 }
 
-#[derive(Serialize, Deserialize, PartialEq, Debug)]
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
 enum PaymentDetails {
-    #[serde(rename = "creditCard")]
     CreditCard(CreditCardDetails),
-    #[serde(rename = "bankAccount")]
     BankAccount(BankAccountDetails),
-    Wallet,
-    Klarna,
```

Comment:
why are other PaymentDetails removed? aren't they supported now?

Reply:
Other payment methods are not implemented for this connector. They don't support this payment methods